routes/index.js

```diff
@@ -39,20 +33,35 @@ apiRouter.use(async (req, res, next) => {
   }
 })
 
-const productsRouter = require('./products'); 
+apiRouter.use((req,res,next) => {
+  if (req.user) {
+    console.log("User is set: ", req.user);
+  }
+
+  next();
+});
+
+const productsRouter = require('./products');
 apiRouter.use('/products', productsRouter);
 
 const usersRouter = require('./users');
 apiRouter.use('/users', usersRouter);
 
-server.use((req, res, next) => {
+const ordersRouter = require('./orders');
+apiRouter.use('/orders', ordersRouter);
+
+const orderProductsRouter = require('./order_products');
+apiRouter.use('/order_products', orderProductsRouter);
+
+/*server.use((req, res, next) => {
```

Comment:
REQUIRED - No commented-out code.  If it should be commented in, un-comment.  Otherwise, remove.